src/oncall/scheduler/default.py

```diff
@@ -53,10 +53,19 @@ def get_busy_user_by_event_range(self, user_ids, team_id, events, cursor):
         for e in events:
             range_check.append('(%r < `end` AND `start` < %r)' % (e['start'], e['end']))
 
+        cursor.execute('''SELECT `subscription_id`, `role_id`
+                          FROM `team_subscription`
+                          WHERE `team_id` = %s''',
+                       team_id)
+        subscriptions = cursor.fetchall()
+        team_check = ['team_id = %s']
+        for sub in subscriptions:
+            team_check.append('(team_id = %r AND role_id = %r)' % (sub['subscription_id'], sub['role_id']))
```

Comment:
I don't think we should rely on immediate `%r` for value interpolation and should instead do %s and maintain a list of values to add to the cursor.execute params